content/articles/getting-started-with-nessus-scanner/index.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /getting-started-with-nessus-scanner/
+title: Getting Started with Nessus Scanner
+description: Nessus scanner is the most popular and widely used vulnerability scanner common among penetration testers and hackers. It is open-source and it allows free scan for up to 16 Private Addresses at a time. 
+author: adeshina-peter
+date: 2022-01-27T00:00:00-17:00
+topics: [Security]
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/getting-started-with-nessus-scanner/hero.jpg
+    alt: Getting Started with Nessus Scanner Hero Image
+---
+**Nessus Tenable** is a kali-Linux open-source software common among penetration testers, used to provide a variety of security fixes. You are going to be using this tool quite frequently while you work as a pen-tester by scanning for potential vulnerability on a Server/Website.
+<!--more-->
+**Nessus** scanner is a professional tool that is widely used among hackers and penetration testers. The solution you get assists you to perform high-speed asset discovery, target profiling, malware detection, sensitive data discovery and so many more.
+**Tenable** delivers updated plug-ins that provide you with the latest information to customers within 24 hours after a vulnerability has been leaked.
+**Nessus Tenable** grant you the privileges to scan basic networks and servers for free, and also provide you with how **critical**,**medium** or how **low** a vulnerability is and helps provide descriptions and solutions for a fix.
+
+Most sites that talk about Nessus only point out how Nessus is being installed and how it is been used, But how to fix a real vulnerability is hardly in the scope of what they teach. I will show you how to fix a common vulnerability and also guide you through important Nessus keywords.
+
+### Goal
+At the end of this tutorial, you should be able to:
+
+- Download and install Kali-Linux on your pc.
+
+- Understand the fundamentals of Nessus Scanner.
+
+- Understand the advantages of Nessus Scanner.
+
+- Install and Download Nessus Tenable.
+
+- Understand procedures to take in fixing vulnerabilities.
+
+### Advantages of Nessus Scanner
+- It helps you Identify vulnerabilities before attackers can take advantage of them.
+
+- It helps you find misconfigurations like missing patches, mail-relay, etc.
+
+- Capturing of data and showing results reporting impact on the network 
+
+If you don't have Kali-Linux installed on your pc, Don't worry I will show you how to quickly set this up.
```

Comment:
Avoid using the word, "I" in your tutorials. The focus should always be on the reader. Please use 'we' instead and add commas where necessary. i.e, don't worry, .......